.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report or request for help
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Version information:**
+ - Tableau Server version: (or note if using Tableau Online)
```

Comment:
Can also be desktop, so this should be
- Tableau version: (or note if using Tableau Online)